pages/glossary/clearly-labeled-location.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Clearly labeled location
+key: clearly-labeled-location
+unambiguous: true
+objective: false
+---
+
+Secondary information and alternative controls or functionality are often not displayed together with primary information or functionality. For example, an option to change a web page to dark mode may be placed on an options page instead of being available on every page and page state of a website. Another example is a maps application, where, instead of using GPS, an option is available in a dropdown menu to set the current location of the device. Such content should be placed in a clearly labeled location.
+
+The location of a target is said to be **"clearly labeled"** when the target can be found by activating [user interface components][] whose purpose, indicated by their [label], leads the user to find the target or to another [page][web page] or page state from which this action can be repeated until the target is found.
+
+Whether or not the content is "clearly labeled" depends on the starting point of the search. If page A has a clearly labeled link to some piece of important content, it is clearly labeled. Page B, which can be in the same website, may not have such a link or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labeled.
+
+**Note:** A [web page][] changes state when the [document's body][body] changes without a change in the [document's URL][URL].
```

Comment:
Do change in the query parameters make a different web page?

Reply:
According to our definition of a web page, I would argue yes because when you change the URL you have a different document, therefore a different fully active document, which would make it a different web page.